llvm/include/llvm/Frontend/Driver/CodeGenOptions.h

```diff
@@ -36,7 +36,8 @@ enum class VectorLibrary {
 };
 
 TargetLibraryInfoImpl *createTLII(llvm::Triple &TargetTriple,
-                                  VectorLibrary Veclib);
+                                  VectorLibrary Veclib,
+				  bool targetAVX512);
```

Comment:
Please use spaces instead of tabs for indenting your code. Try running `git clang-format` on your diff.